package.json

```diff
@@ -3,7 +3,8 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "@proton/web-sdk": "2.7.13",
+    "@proton/js": "22.0.45",
```

Comment:
Proton Js is not a package that was originally used here. There's no need to add it if it wasn't originally used. Can you try removing it to see if this app will still work?